src/common/libflux/message.c

```diff
@@ -277,11 +277,15 @@ void *flux_msg_aux_get (const flux_msg_t *msg, const char *name)
     return aux_get (msg->aux, name);
 }
 
-size_t flux_msg_encode_size (const flux_msg_t *msg)
+ssize_t flux_msg_encode_size (const flux_msg_t *msg)
```

Comment:
Probably should update the one user in librouter/sendfd.c to handle the error case that's now possible.